CONTRIBUTING.md

```diff
@@ -279,9 +279,8 @@ For instance:
    and see if the CI pipeline passes (this will both check binary compatibility and run UI tests against the given EAP)
 4. we release the plugin as `0.8.0` (`untilBuild` will extend automatically to `2020.3.*`
    via `latestSupportedMajor` in [IntellijVersions](buildSrc/src/main/kotlin/com/virtuslab/gitmachete/buildsrc/IntellijVersions.kt))
-5. once the stable `2020.3` is released, we verify ASAP that `0.8.0` is binary compatible with `2020.3` as well;
-   then, `latestStable` can be updated to `2020.3`, `eapOfLatestSupportedMajor` can be set to `null`,
-   and `2020.2.x` can be added to `latestMinorsOfOldSupportedMajors`
+5. new stable version can be detected automatically using `./gradlew updateIntellijStableVersions`,
```

Comment:
Pls update the entire section, assuming as for now that the project will be left on _autopilot_, i.e. all updates to intellijVersions.properties will be (ideally) done automatically by the bot, and never by the developers. It'll still be useful to document the logic&rationale behind these updates, however